src/multio/action/interpolate/FesomCacheGenerator.cc

```diff
@@ -146,6 +146,8 @@ Fesom2mirCacheGenerator::Fesom2mirCacheGenerator(int argc, char** argv) :
     options_.push_back(new eckit::option::SimpleOption<std::string>(
         "inputFile", "Name of the input file. Default( \"CORE2_ngrid_NSIDE32_0_ring.csv\" )"));
     options_.push_back(new eckit::option::SimpleOption<size_t>("nCols", "Size of the fesom grid."));
+    options_.push_back(new eckit::option::SimpleOption<std::string>(
```

Comment:
Why do we need the name in addition to, say, the input file?

Reply:
You're right I was misled by the default option as core2 and wanted to update that before telling Sebastian. I didn't see the post where regex is used to create the fesomName. I can delete this pr